apps/ledger-live-desktop/src/renderer/modals/Send/types.ts

```diff
@@ -12,7 +12,7 @@ export type StepProps = {
   device: Device | undefined | null;
   account: AccountLike | undefined | null;
   parentAccount: Account | undefined | null;
-  transaction: Transaction | undefined | null;
+  transaction: (Transaction & { memo?: string }) | undefined | null;
```

Comment:
Is this necessary ? If you use `"memo" in transaction` typescript would infer the right union type for the Transaction (something like "cardanoTransaction" | "cosmosTransaction" | ...). This removes this check and might prevent some issue to be caught (e.g in Solana the field is also called memo but it exist somewhere else in the tx structure (at least in LLM). WDYT ?

Reply:
well it's not, while `Transaction` type is a union of types (not the mathematical union) , accessing to memo properties will leads to a typeError saying that memo doesn't exist in `Transaction` type, that's why on this specific scope i preffered a local explicit typing. At this stage, i won't change the original types (built 3 years ago) as it's not in the scope of this ticket and it's more technical task than functional one.